### PR TITLE
add support for docker

### PR DESCRIPTION
**Pull request by edsu7**

### CODE REVIEW

**Review by lindaxiang — CHANGES_REQUESTED** [review]

File: process/score_upload.nf [hunk 1]
```diff
@@ -35,7 +35,9 @@ process scoreUpload {
     container "${ params.score_container ?: params.container}:${params.score_container_version ?: params.container_version}"
 
     if (workflow.containerEngine == "singularity") {
-        containerOptions "--bind \$(pwd):/score-client/logs"
+        containerOptions "--bind \$(pwd):/song-client/logs"
+    } else if (workflow.containerEngine == "docker") {
+        containerOptions "-v \$(pwd):/song-client/logs"
```
[the remark this answers is not in the document]
Comment (edsu7) [reply process/score_upload.nf]: Fixed in
https://github.com/icgc-argo-workflows/nextflow-data-processing-utility-tools/blob/1a01b2e4e0f1acbee0fa08d6016ab5702d8bf818/process/score_upload.nf#L37-L40

File: process/score_upload.nf [hunk 2]
```diff
@@ -36,6 +36,8 @@ process scoreUpload {
 
     if (workflow.containerEngine == "singularity") {
         containerOptions "--bind \$(pwd):/score-client/logs"
+    } else if (workflow.containerEngine == "docker") {
+        containerOptions "-v \$(pwd):/score-client/logs"
```
Comment (lindaxiang) [inline process/score_upload.nf:40]: A very minor suggestion, can we change to 
```
-v \$(pwd)/logs:/score-client/logs
```

**Review by lindaxiang — APPROVED** [review]
- A very minor suggestion. 
- Question: do we plan to make similar changes to `song-score-download`?

**Comment by edsu7** [comment]
closing. new branch in https://github.com/icgc-argo-workflows/nextflow-data-processing-utility-tools/pull/51

**Pull request closed without merging**